src/rgw/rgw_lc.cc

```diff
@@ -2188,6 +2188,7 @@ std::string s3_expiration_header(
 
     if (filter.has_tags()) {
       bool tag_match = false;
+      size_t tag_count = 0;
```

Comment:
can't we just ensure that the tag enumeration detects that a tag was not matched?

Reply:
Done